app/components/UserAttendance/AttendanceModalContent.tsx

```diff
@@ -0,0 +1,144 @@
+import cx from 'classnames';
+import { flatMap } from 'lodash';
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import Button from 'app/components/Button';
+import { TextInput } from 'app/components/Form';
+import { ProfilePicture } from 'app/components/Image';
+import Flex from 'app/components/Layout/Flex';
+import type { ID } from 'app/store/models';
+import type { PublicUser } from 'app/store/models/User';
+import styles from './AttendanceModalContent.css';
+
+export type Registration = {
+  id: ID;
+  user: PublicUser;
+};
+
+export type Pool = {
+  name: string;
+  registrations: Registration[];
+};
+
+type Props = {
+  pools: Pool[];
+  title: string;
+  togglePool: (index: number) => void;
+  selectedPool: number;
+  isMeeting?: boolean;
+};
+
+type TabProps = {
+  name: string;
+  index: number;
+  activePoolIndex: number;
+  togglePool: (pool: number) => void;
+};
+
+const Tab = ({ name, index, activePoolIndex, togglePool }: TabProps) => (
+  <Button
+    flat
+    onClick={() => togglePool(index)}
+    className={cx(
+      styles.navButton,
+      activePoolIndex === index && styles.activeItem
+    )}
+  >
+    {name}
+  </Button>
+);
+
+const AttendanceModalContent = ({
+  title = 'Status',
+  pools,
+  togglePool,
+  selectedPool,
+  isMeeting,
+}: Props) => {
+  const [amendedPools, setAmendedPools] = useState<Pool[]>([]);
+  const [registrations, setRegistrations] = useState<Registration[]>([]);
+  const [filter, setFilter] = useState<string>('');
+
+  useEffect(() => {
+    generateAmendedPools(pools);
+  }, [pools]);
+
+  const generateAmendedPools = (pools: Pool[]) => {
+    if (pools.length === 1) return setAmendedPools(pools);
+
+    const registrations = flatMap(pools, (pool) => pool.registrations);
+    const summaryPool = {
+      name: 'Alle',
+      registrations,
+    };
+    return setAmendedPools([summaryPool, ...pools]);
+  };
```

Comment:
Not related to this change directly but since you are cleaning up.. :grimacing: 

This should be a `useMemo` and the `generateAmemdedPools` should be a pure function defined outside the component. Should be faster, alot cleaner and more reacty :)